### PR TITLE
Add simple run-example in readme

### LINKED ISSUE #45: Document use and startup of API-application
Should be added to the README.md

### PR DESCRIPTION
**Pull request by johtoblan**
**Summary**: Adds documentation how to send a file with curl to the insert endpoint

**Related issue**: Closes #45

**Suggested reviewer(s)**: @aheimsbakk 

**Reviewer checklist**:

* [ ] The headers of all files contain a reference to the repository license
* [ ] 100% test coverage of new code - meaning:
  * [ ] The overall test coverage increased or remained the same as before
  * [ ] Every function is accompanied with a test suite
  * [ ] Tests are both positive (testing that the function work as intended with valid data) and negative (testing that the function behaves as expected with invalid data, e.g., that correct exceptions are thrown)
  * [ ] Functions with optional arguments have separate tests for all options
* [ ] Examples are supported by doctests
* [ ] All tests are passing
* [ ] All names (e.g., files, classes, functions, variables) are explicit
* [ ] Documentation (as docstrings) is complete and understandable

The checklist is based on the S-ENDA conventions and definition of done (see [General Conventions](https://s-enda-documentation.readthedocs.io/en/latest/general_conventions.html)). The above points are not necessarily relevant to all contributions. In that case, please add a short explanation to help the reviewer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,21 @@ To increase logging level to include info and debug messages, set the environmen
...
+```python
+python dmci_start_api.py
+```
+
+Then you can post with curl to the api - endpoint:
+
+```bash
+curl --data "@<PATH_TO_MMD_FILE>" localhost:5000/v1/insert
+```
+insert is currently the only implemented command. Other commands will return
```
Comment (aheimsbakk) [inline README.md:70]: Return? What? :)

What about the different return codes 200/400 and so on?

**Review by aheimsbakk — COMMENTED** [review]
Added one comment.

File: README.md (hunk 1 above)
Comment (johtoblan) [reply README.md:70]: Oh, thanks I'll fix.

Comment (aheimsbakk) [reply README.md:70]: Was there a status page i could look at too? Healtz or something similar?

**Review by vkbo — APPROVED** [review]

**Pull request merged**